Cargo.toml

```diff
@@ -25,10 +25,10 @@ cors = ["tide-cors"]
 hyper = ["http-service-hyper"]
 
 [dependencies]
-futures-preview = "0.3.0-alpha.17"
+futures-preview = "0.3.0-alpha.18"
 http = "0.1"
-http-service = "0.3.1"
-http-service-hyper = { version = "0.3.1", optional = true }
+http-service = { git = "https://github.com/rustasync/http-service", branch = "tokio-0.2" }
+http-service-hyper = { git = "https://github.com/rustasync/http-service", branch = "tokio-0.2", optional = true }
```

Comment:
Is this still needed? I believe we already published updated versions right?

Reply:
If you look closely, this actually depends on the branch. Specifically, this one:

rustasync/http-service#43

Reply:
`0.3.1` was supposed to release earlier, but I only got it done today after seeing https://github.com/rustasync/http-service/issues/45

Is there a reason we aren't using overrides for the Git dependencies in this branch? (Seems that would prevent issues with publishing, but correct me if I'm wrong.)

Reply:
Not sure -- Git deps can't be published to crates.io either way.